pkg/cmd/controller/interfaces.go

```diff
@@ -38,9 +39,9 @@ import (
 )
 
 func NewControllerContext(
+	ctx context.Context,
 	config openshiftcontrolplanev1.OpenShiftControllerManagerConfig,
 	inClientConfig *rest.Config,
-	stopCh <-chan struct{},
```

Comment:
This bit of refactoring is needed to run the ServiceAccount controller. As a follow up, the other controllers should be updated to be initialized with this context.